docs/src/gs/create-workload-cluster.md

```diff
@@ -2,66 +2,81 @@
-| `CONTROL_PLANE_MACHINE_COUNT`  |       1                | The number of control plane machines for the workload cluster.|
-| `KUBERNETES_VERSION`           |                        | The Kubernetes version to use for the workload cluster. If unspecified, the value from OS environment variables or the .cluster-api/clusterctl.yaml config file will be used. |
-| `NAMESPACE`                    |                        | The namespace to use for the workload cluster. If unspecified, the current namespace will be used |
-| `POD_CIDR`                     |       1                | The CIDR range for the Kubernetes POD network. |
-| `SERVICE_CIDR`                 |                        | The CIDR for the Kubernetes services network.  |
-| `SERVICE_DOMAIN`               |                        |  |
-| `WORKER_MACHINE_COUNT`         |                        | The number of worker machines for the workload cluster. |
+| Parameter                     | Default Value  | Description                                                                                                                                                                   |
+|-------------------------------|----------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `CLUSTER_NAME`                |                | The name of the workload cluster to create.                                                                                                                                   |
+| `CONTROL_PLANE_MACHINE_COUNT` | 1              | The number of control plane machines for the workload cluster.                                                                                                                |
+| `KUBERNETES_VERSION`          |                | The Kubernetes version to use for the workload cluster. If unspecified, the value from OS environment variables or the .cluster-api/clusterctl.yaml config file will be used. |
+| `NAMESPACE`                   |                | The namespace for the workload cluster. If not specified, the current namespace is used.                                                                                      |
+| `POD_CIDR`                    | 192.168.0.0/16 | CIDR range of the Kubernetes pod-to-pod network.                                                                                                                              |
+| `SERVICE_CIDR`                | 10.128.0.0/12  | CIDR range of the Kubernetes pod-to-services network.                                                                                                                         |
+| `SERVICE_DOMAIN`              |                |                                                                                                                                                                               |
```

Comment:
Why is there no description for this value? I didn't notice this in the previous review, but we shouldn't leave this undefined. If this is referring to the DNS domain name, it would be better to use `SERVICE_FQDN`, i.e. fully qualified domain name. 

Either way, it should have a description.

Reply:
The description still doesn't actually tell me what this field is for or what it actually does. It's just the name of the field in prose format.

Reply:
@joekr please remove this field fromt he templates and description, other providers (aws/azure) dont have the field in their templates and from a look at the code, the field is not the right place to override Kubernetes service domain name.